test/issue1396-pre-post-run-commands.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+. $(dirname "${BASH_SOURCE[0]}")/common.sh
+cd ${CURR_DIR}/issue1396-pre-post-run-commands
+rm -rf .dub
```

Comment:
Isn't this dangerous when the testsuite is running in parallel (though that's not really supported as of now)

Reply:
Depending how the test suite parallel runner works this fails or not. I am in doubt to add here a more complex logic which might be needed or not.